persistent/picklecache.py

```diff
@@ -156,18 +229,20 @@ def klass_items(self):
     def incrgc(self, ignored=None):
         """ See IPickleCache.
         """
-        target = self.target_size
+        target = self.cache_size
```

Comment:
This change doesn't seem correct:  `self.target_size` is the configured value to which the cache is minimized.